docs/getting_started/ibexa_cloud_guide.md

```diff
@@ -0,0 +1,111 @@
+This cloud computing approach gives users access to a full cloud platform, including hardware, software, and infrastructure.
+PaaS eliminates the requirement to buy and install the necessary hardware and software. All you have to do is access it and you can start deploying resources and developing right away.
+
+![How PaaS works](how_paas_works.png)
+
+### Automation
+
+[[= product_name_cloud =]] automates time-consuming testing processes and encourages ongoing QA testing, so your projects are ready for deployment much faster.
+For further protection and easy rollbacks, it also automates your backup procedures.
+This guarantees that the framework developers are working within is adaptable, safe, and responsive to their requirements.
+Thanks to built-in CI/CD features, you can also reduce the need for manual testing and accelerate.
+
+### Digital tool
+
+[[= product_name_cloud =]] is a digital tool - it's a perfect starting step toward your company's digital transformation.
+What's more, [[= product_name_cloud =]] as an end-to-end cloud hosting platform, it's scalable, requires little up-front expenditure, and no ongoing maintenance.
+[[= product_name_cloud =]] as a PaaS solution enables you to start constructing effective digital systems that provide great consumer experiences.
+
+### Disk space
+
+Your [[= product_name_cloud =]] subscription level determines the overall amount of disk space available.
+You can assign disk space to the main app container under the disk key.
+You can also divide the remaining space between additional containers (such as the database) or the search engine in `platform.sh` configuration file, under the specific service definitions.
+
+## Benefits
+
+### Safety and simplicity
+
+Comprehensive data security procedures guarantee that you maintain complete ownership of your client's data, defining where it's stored.
+You can be sure that the safety protocols are compliant with all applicable legislation.
+What's more, all updates to your code and infrastructure are fully auditable. Global CND (Content Delivert Network) is included and fully managed.
+[[= product_name_cloud =]], thanks to grid architecture and extensive [[= product_name_base =]] support, enables easy deployment.
+
+### High availability and compatibility
+
+[[= product_name_cloud =]] is compatible with your choice of public cloud server and support for hosting platforms.
+It's a Git-native development - compatible with Git Flow. You can also integrate with, for example, Bitbucket, GitHub, Gitlab.
+You can instantly clone every branch of both your code and infrastructure.
+If you want to work with services such as MySQL, Kafka, and Elasticsearch, you can simply add them with a line of code.
+What's more, you can run in your chosen cloud, like Microsoft Azure, Orange, Google Cloud Platform, and regional partners.
+
```

Comment:
In this section, we could nail that it will integrate perfectly with the probable internal usage of Git.
Something like:
```suggestion
As you probably already use Git internally for team work and change tracking, [[= product_name_cloud =]] deployment integrate naturally at the end of your existing production chain, including staging, and work in progress branch preview.

```

Reply:
Nicely added in 7d226d72ff24ef0c331dda61013ca1874f7ea9ba